hw3/server/src/index.ts

```diff
@@ -0,0 +1,71 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { Transform } from 'stream';
+import { EventData } from './types/EventData';
+
+// Exercise №1
+
+const inputFilePath = path.join(__dirname, 'docs', 'input.txt');
```

Comment:
fix that please 
[Error: ENOENT: no such file or directory, open '/Users/ivankornev/Desktop/Students HW/Gleb Kovalchuk/Hillel-HW-NodeJS/hw3/server/src/docs/input.txt'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/ivankornev/Desktop/Students HW/Gleb Kovalchuk/Hillel-HW-NodeJS/hw3/server/src/docs/input.txt'
}
[nodemon] app crashed - waiting for file changes before starting...